src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -14,6 +14,9 @@
 
   private int torpedoCount = 0;
 
+  // simulate random overheating of the launcher bay which prevents firing
+  Random generator = new Random();
```

Comment:
Using a simple Random and not a SecureRandom isn't a problem for SonarCloud quality gate check?